main.tf

```diff
@@ -148,7 +149,7 @@ module "autoscale_group" {
 
   image_id                  = "${var.use_custom_image_id == "true" ? var.image_id : join("", data.aws_ami.eks_worker.*.id)}"
   iam_instance_profile_name = "${local.use_existing_instance_profile == "false" ? join("", aws_iam_instance_profile.default.*.name) : var.aws_iam_instance_profile_name}"
-  security_group_ids        = ["${join("", aws_security_group.default.*.id)}"]
+  security_group_ids        = ["${compact(concat(local.use_existing_workers_security_group == "false" ? join("", aws_security_group.default.*.id) : var.workers_security_group_id}", "${var.additional_security_group_ids}))"]
```

Comment:
```suggestion
    security_group_ids        = ["${compact(concat(list(local.use_existing_workers_security_group == "false" ? join("", aws_security_group.default.*.id) : var.workers_security_group_id), var.additional_security_group_ids))}"]

```

Reply:
please check the CI/CD for errors
https://travis-ci.org/cloudposse/terraform-aws-eks-workers/builds/529627667?utm_source=github_status&utm_medium=notification